### PR TITLE
Add "How open source licenses work and how to add them to your projects" to list

### PR DESCRIPTION
**Pull request by notsu**
I added two resources which will be useful for the programmer.

Security Resources
- [A collection of hacking/penetration testing resources to make you better!](https://github.com/vitalysim/Awesome-Hacking-Resources)

Licenses
- [How open source licenses work and how to add them to your projects](https://medium.freecodecamp.org/how-open-source-licenses-work-and-how-to-add-them-to-your-projects-34310c3cf94)

Both of them will help every programmer improve their skill better and kickstart their own opensource.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,6 +168,7 @@ But knowing the stuff will help you become better! :muscle:*
 ### Licenses
 - [Choose An Open Source License](https://choosealicense.com/)
 - [Well-explained Software licenses in TLDR version](https://tldrlegal.com/)
+- [How open source licenses work and how to add them to your projects](https://medium.freecodecamp.org/how-open-source-licenses-work-and-how-to-add-them-to-your-projects-34310c3cf94)
```
Comment (eamanu) [inline README.md]: Great idea

File: README.md [hunk 2]
```diff
@@ -75,6 +75,7 @@ But knowing the stuff will help you become better! :muscle:*
 - :book: [Foundations of Security: What Every Programmer Needs to Know ](https://www.goodreads.com/book/show/128003.Foundations_of_Security)
 - [OWASP Top 10](https://www.owasp.org/index.php/Category:OWASP_Top_Ten_Project)
 - [Web Application Exploits and Defenses](https://google-gruyere.appspot.com/part1)
+- [A collection of hacking / penetration testing resources to make you better!](https://github.com/vitalysim/Awesome-Hacking-Resources)
```
Comment (eamanu) [inline README.md]: I don't know if hacking info is relevant in this repo. Maybe you have to add this in [Awesome Hacking Resource](https://github.com/vitalysim/Awesome-Hacking-Resources)

Comment (notsu) [reply README.md]: It's linked to that repo which you referred to.

But I'm not sure about programmer should know some about hacking before prevent it? :sweat_smile:

**Comment by mr-mig** [comment]
@notsu I would ask you to split this PR and leave just "How open source licenses work and how to add them to your projects". 

I would not add Hacking resources for now, as those are quite specific

**Comment by notsu** [comment]
@mr-mig removed that link, please check again.

**Comment by siriniok** [comment]
Thank you! Squashed and merged.

**Pull request merged**